### DISCUSSION TITLE
Design patterns explained with sketches

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#28] [General]
# Design patterns explained with sketches

This article will explain design patterns, which we use on a daily base, with smaller (over)simplified illustrations.

https://steven-giesel.com/blogPost/91629412-1440-4aa6-add8-acd925831b35

**Comment by axilis-marko** [comment]
In chain of responsibility there is a typo: `IT suppoirt`.

**Reply by linkdotnet** [reply]
Thanks for reporting @axilis-marko. The fixed and new version is live (with some other smaller improvements).

**Comment by mushifali** [comment]
There is a typo in strategy pattern: "surf _[sic]_ the same purpose".

**Reply by linkdotnet** [reply]
As I am not a native speaker, which version would be preferred?

**Reply by axilis-marko** [reply]
`serve`. `surf` is to go out on to the ocean and ride the waves 😄

**Reply by linkdotnet** [reply]
Daahhh.. so obvious :D thanks for pointing that out. @axilis-marko . I corrected it and uploaded a fixed version.

**Comment by toupswork** [comment]
i like it but wish it had more patterns

**Reply by linkdotnet** [reply]
I did this pre AI :) not sure if there would be still value.
I initially expanded some, but stopped as (for my feeling) I got the major ones.
It is never too late to pick it up again :)
